### PR TITLE
Updated 'Dockerize a Svelte App' as it was repeating itself a little.

### PR DESCRIPTION
**Pull request by cainux**
Also reworded some parts to help explain what is going on.

### CODE REVIEW

**Comment by swyxio** [comment]
nice! needs someone who knows docker to review this... @antony ?

**Comment by antony** [comment]
Will review asap :) Thanks @cainux

File: src/pages/recipes/publishing-and-deploying/dockerize-a-svelte-app.svx [hunk 1]
```diff
@@ -25,29 +27,21 @@ FROM nginx:1.19-alpine
...
-COPY . .
+### Running
 
-RUN npm install
-RUN npm run build
+And to run your image:
 
-FROM nginx:stable
+```
+docker run --rm --name=svelte-docker -p 5000:80 svelte-docker
```
Comment (antony) [inline src/pages/recipes/publishing-and-deploying/dockerize-a-svelte-app.svx:44]: My comment here (and I realise the original did it too) is why we bother binding to port 5000? Certainly on Linux, if you're running docker as root, or have added yourself to the docker group, you can bind to 80. I don't know if it's the same on recent Mac/Windows, as I know they run Docker in a virtual machine or some other nastiness.

**Review by antony — APPROVED** [review]
Looks like a good cleanup!

File: src/pages/recipes/publishing-and-deploying/dockerize-a-svelte-app.svx (hunk 1 above)
Comment (cainux) [reply src/pages/recipes/publishing-and-deploying/dockerize-a-svelte-app.svx:44]: I haven't used a Mac in a while but I know in Windows the default configuration won't let you bind to port 80.

**Pull request merged**